charts/prometheus-thanos/values.yaml

```diff
@@ -41,6 +41,8 @@ querier:
     repository: quay.io/thanos/thanos
     tag: v0.10.0
     pullPolicy: IfNotPresent
+  serviceAccount:
+    create: true
```

Comment:
Please set this value by default to `false`.
I guess it is worth to add an example config (as comment) for other users how to use the mentioed IAM feature (espacially the related annotations and a reference link to AWS or GCP docu)

Reply:
I added an exemple and a link to AWS doc in values.yaml